lib/util/chat/pages/chat_page.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:flutter/material.dart';
+import 'package:motis_mitfahr_app/account/models/profile.dart';
+import 'package:motis_mitfahr_app/util/chat/message_bar.dart';
+import 'package:motis_mitfahr_app/util/chat/models/message.dart';
+import 'package:motis_mitfahr_app/util/chat/chat_bubble.dart';
+import 'package:motis_mitfahr_app/util/locale_manager.dart';
+import 'package:motis_mitfahr_app/util/profiles/profile_widget.dart';
+import 'package:motis_mitfahr_app/util/supabase.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class ChatPage extends StatefulWidget {
+  const ChatPage({required this.rideId, required this.profile, super.key});
+  final int rideId;
+  final Profile profile;
+
+  @override
+  State<ChatPage> createState() => _ChatPageState();
+}
+
+class _ChatPageState extends State<ChatPage> {
+  late Stream<List<Message>> _messagesStream;
+
+  @override
+  void initState() {
+    _messagesStream = SupabaseManager.supabaseClient
+        .from('messages')
+        .stream(primaryKey: ['id'])
+        .eq('ride_id', widget.rideId)
+        .order('created_at')
+        .map((messages) => Message.fromJsonList(messages));
+    super.initState();
+  }
+
+  @override
+  void dispose() {
+    _messagesStream.drain();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: ProfileWidget(
+          widget.profile,
+          isTappable: true,
+        ),
+      ),
+      body: StreamBuilder<List<Message>>(
+        stream: _messagesStream,
+        builder: (context, snapshot) {
+          if (snapshot.hasData) {
+            List<Message> messages = snapshot.data!;
+            return Column(
+              children: [
+                Expanded(
+                  child: messages.isEmpty
+                      ? Column(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          children: [
+                            Image.asset(
+                              'assets/chat_shrug.png',
```

Comment:
Maybe we should just use the same picture as in search?

Reply:
yeah i wasn't sure about that, what do the others think?

Reply:
But it's confusing and a different art style

Reply:
I agree with @istzen, I think it kind of adds personality to the app if we have some different pictures on different pages, so that it's kind of an easter egg to find all of them. (Not that I want to fill our **PROFESSIONAL BUSINESS APP** with tight time constraints with easter eggs, but that's the vibe I'm getting from this, especially if we also do this with other lists. Otherwise I'm getting the vibe of "Oh their design studio was paid per image".)
Also, wasn't Merri REALLY looking forward to drawing some beautiful pictures for our app @istzen ;)?

Reply:
Why is it a bad thing if we use a consistent image to signify the same concept? "No results". Maybe if the image here would somehow include the fact that we are talking about chats, then I would agree. But now it's basically the same image, but in a different style.